src/cls/IPM/Main.cls

```diff
@@ -1767,6 +1768,9 @@ ClassMethod Repository(ByRef pCommandInfo) [ Internal ]
 		}
 		While tRepoTypeResult.%Next(.tSC) {
 			$$$ThrowOnError(tSC)
+			If tRepoTypeResult.%Get("Moniker") = "http" {
```

Comment:
It's because the class hierarchy. Both _Remote_ and _Oras_ repos are subclasses of _Http_ and _Http_ itself is not an abstract class. Normally, _Http_ repos will never be created because the zpm `repo` command doesn't allow for such an operation. However, if something goes wrong or if a user manually created a instance of this type, then we need to ignore it (or even raise a warning/error).